frontend/src/__tests__/cypress/cypress/pages/pipelines/pipelinesGlobal.ts

```diff
@@ -26,6 +26,11 @@ class PipelinesGlobal {
   selectProjectByName(name: string) {
     this.findProjectSelect().findDropdownItem(name).click();
   }
+
+  findDeleteButton() {
+    this.find().findByLabelText('More actions').click();
```

Comment:
I add to chnage this to cy.get instead of this.find() in my ux fix PR becuas rthe lowest i could add a test id without issue was at the table level, which doesnt include this toolbar button

Reply:
I don't think it's necessary to scope any page level locators. So a `get` is find for _all_ locators of a page object. Unless we want to take page scoping one step further and make them scoped to the PF layout container of the main content.

Reply:
right. i think i just meant use `cy` so `cy. findByLabelText(...)`